server.py

```diff
@@ -91,8 +91,7 @@ def __init__(self) -> None:
         self.LastBoosters = int(time.time() * 1000)
         self.TurnSpeed = 0.05
         self.Speed = 0.03
-        self.StartPositions = [0, 180, 90, 270, 45, 225, 135, 315, 0, 200, 110, 290, 340, 160, 70, 250, 225, 320,
-                               1, 5, 10, 15, 20, 25, 30, 35, 35, 35, 35, 35, 35, 35, 35, 35]
+        self.StartPositions = [0, 180, 90, 270, 45, 225, 135, 315, 0, 200, 110, 290, 340, 160, 70, 250, 225, 320]
```

Comment:
Probably it is better to use formula to calculate position, instead of array of position.